try it update (#23)
wso2/docs-ciam-cloud: en/docs/guides/organization-login/try-organization-login.md

!!! note "Create the users"
    Create new users on the sub-organizations with the required permissions of an administrator.
    
    - To create a user for **Best Auto Mart**:
        1. Use the **Organization Switcher** to change the organization to **Best Auto Mart**.
        2. Create a user named `Alex` on the **Best Auto Mart** organization.
        3. Create a **Role** with the [required permissions](../../b2b-org-management/b2b-org-permissions) to create an Identity Provider. Assign `Alex` to this newly created **Role**.
    - To create a user for **Car Traders**:
        1. Use the **Organization Switcher** to change the organization to **Car Traders**.
        2. Create a user named `Sam` on the **Car Traders** organization.
        3. Create a **Role** with the [required permissions](../../b2b-org-management/b2b-org-permissions) for an administrator. Assign `Sam` to this newly created **Role**.

## Step 2: Configure the business apps

To configure the business applications:

1. On the CIAM Private Cloud console, go to **Develop > Applications**.
2. Click **+ New Application** and select **Standard-Based Application**.

> 3. Enter `Guardio-SaaS-app` as the application name and select `OAuth2 / OpenID Connect` as the **Protocol**.

Reviewer: ```suggestion
3. Enter `Guardio-SaaS-app` as the application name and select `OAuth2.0 / OpenID Connect` as the **Protocol**.
```

6. On the Protocol tab, select [`Organization Switch`](../../../references/org-domains-urls) and `Code` on the **Allowed Grant types**, and enter the following details:
    - Authorized redirect URLs: `http://localhost:3000/api/auth/callback/wso2is`
    - Allowed origin: `http://localhost:3000`

7. Click **Update** to save the configurations.

## Step 3: Share the business app

Share the `Guardio-SaaS-app` business application with the other sub-organizations using the [Share Application](../organization-login/share-the-business-app/#share-the-application) option.

<img src="../../../assets/img/guides/organization-login/try-it-out/share_app_with_sub_orgs.png" alt="Share App with suborganizations" width="600" style="border:1px solid grey">

## Step 4: Configure the Sign-in method

After you share the application with the sub-organizations, an **Organization SSO** IdP named `Organization Login` will be automatically created and assigned as a sign-in method for the application.

To check if the IdP is assigned to the application:

1. On the CIAM Private Cloud console, go to **Develop > Applications** and select `Guardio-SaaS-App`.

> 2. On the **Sign-in Method** tab, check if the **Organization Login** IdP has been assigned.

Reviewer: ```suggestion
2. On the **Sign-in Method** tab, check if the generated **Organization Login** IdP has been assigned.
```

# Try Organizational Login Flow

Let's try organization login using a sample scenario.

## Scenario

> **Guardio** is an auto insurance service company that has many employees who use different credentials to sign in to many internal applications.  To mange logins to its many applications, Guardio is using the WSO2 Identity Server B2B Private CIAM Solution where Guardio is configured as a super organization. *Guardio-SaaS-App** is one such application. 

Reviewer 2: ```suggestion
**Guardio** is an auto insurance service company that has many employees who use different credentials to sign in to many internal applications.  To manage logins to its many applications, Guardio uses the WSO2 Identity Server B2B Private CIAM Solution where Guardio is configured as a super organization. *Guardio-SaaS-App** is one such application. 
```

    tag_requests = false
    ```

Save the file and restart the Identity Server.

## Try it out

To try out this application:

1. Open a terminal, navigate to `<SAMPLE_APP_HOME>/guardio-insurance-sample-application-main` folder and run the following commands to start the application:
    
    !!! info
        To complete this step, it is required to have a node installed in your machine.
        
    ``` node
    npm install
    npm run dev
    ```

2. Enter `http://localhost:3000` on your browser to access the application.

3. Click **Sign In** to proceed with the application.

4. Select your organization from the list and click **Next**. In this example, we will select `Car Traders` as the organization.

    <img src="../../../assets/img/guides/organization-login/try-it-out/sample_app_org.png" alt="Sample App Orgs" width="700" style="border:0px solid grey">

5. Enter the user credentials and click **Continue** to log in to the application.
    You will be successfully logged in to the sample application.

> 6. Logout of the application, select `Best Auto Mart` organization and try to log in. You will be redirected to the Asgardeo login page.

Reviewer 2: If we are going with the terms "sign in" and "sign out", we should consider using the same throughout the doc.
